subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ConfigurationCacheGradleExecuter.groovy

```diff
@@ -33,7 +33,7 @@ class ConfigurationCacheGradleExecuter extends DaemonGradleExecuter {
     ].collect { it.toString() }
 
     static boolean testWithLoadAfterStore() {
-        return !System.getProperty("org.gradle.configuration-cache.internal.test-disable-load-after-store")
+        return !Boolean.getBoolean("org.gradle.configuration-cache.internal.test-disable-load-after-store")
```

Comment:
It bothered me that `...-disable-load-after-store=false` would still disable (only "" is falsy, "false" is not).